docs/source/guides/models.mdx

```diff
@@ -247,7 +247,40 @@ Now generate an image with a prompt on neuron:
 
 <img
   src="https://raw.githubusercontent.com/huggingface/optimum-neuron/main/docs/assets/guides/models/01-sd-image.png"
-  alt="search ami"
+  alt="stable diffusion generated image"
+/>
+
+## Stable Diffusion XL
+
+Similar to Stable Diffusion, you will be able to use `NeuronStableDiffusionXLPipeline` API to export and run inference on Neuron devices with SDXL models. 
+
+```python
+>>> from optimum.neuron import NeuronStableDiffusionXLPipeline
+
+>>> model_id = "stabilityai/stable-diffusion-xl-base-1.0"
+>>> compiler_args = {"auto_cast": "matmul", "auto_cast_type": "bf16"}
+>>> input_shapes = {"batch_size": 1, "height": 1024, "width": 1024}
```

Comment:
is batch_size <> num_images_per_prompt?

Reply:
here the batch size indicate the batch size of prompt, `num_images_per_prompt` is default to 1 if not given.